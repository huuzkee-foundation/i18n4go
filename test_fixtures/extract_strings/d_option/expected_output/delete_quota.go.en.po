# filename: d_option/input_files/quota/delete_quota.go, offset: 863, line: 31, column: 16
msgid "Delete a quota"
msgstr "Delete a quota"

# filename: d_option/input_files/quota/delete_quota.go, offset: 896, line: 32, column: 16
msgid "CF_NAME delete-quota QUOTA [-f]"
msgstr "CF_NAME delete-quota QUOTA [-f]"

# filename: d_option/input_files/quota/delete_quota.go, offset: 1226, line: 41, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/quota/delete_quota.go, offset: 1589, line: 62, column: 13
msgid "Deleting quota %s as %s..."
msgstr "Deleting quota %s as %s..."

# filename: d_option/input_files/quota/delete_quota.go, offset: 1271, line: 42, column: 27
msgid "delete-quota"
msgstr "delete-quota"

# filename: d_option/input_files/quota/delete_quota.go, offset: 1480, line: 55, column: 13
msgid "f"
msgstr "f"

# filename: d_option/input_files/quota/delete_quota.go, offset: 986, line: 34, column: 35
msgid "Force deletion without confirmation"
msgstr "Force deletion without confirmation"

# filename: d_option/input_files/quota/delete_quota.go, offset: 1522, line: 56, column: 36
msgid "quota"
msgstr "quota"

# filename: d_option/input_files/quota/delete_quota.go, offset: 1879, line: 73, column: 15
msgid "Quota %s does not exist"
msgstr "Quota %s does not exist"

