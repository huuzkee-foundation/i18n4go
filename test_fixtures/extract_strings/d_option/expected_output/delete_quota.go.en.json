[{"id":"Delete a quota","translation":"Delete a quota"},{"id":"CF_NAME delete-quota QUOTA [-f]","translation":"CF_NAME delete-quota QUOTA [-f]"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"Deleting quota %s as %s...","translation":"Deleting quota %s as %s..."},{"id":"delete-quota","translation":"delete-quota"},{"id":"f","translation":"f"},{"id":"Force deletion without confirmation","translation":"Force deletion without confirmation"},{"id":"quota","translation":"quota"},{"id":"Quota %s does not exist","translation":"Quota %s does not exist"}]
